examples/select_clusters.md

```diff
@@ -11,12 +11,9 @@
 
 `client.select_clusters(where='gene', ...)`:
 ```python
->>> clusters_with_gene_set = client.select_clusters(where='gene', has=['CASTOR2'], genomic_modality='atac', p_value=0.05)
+>>> clusters_with_gene_set = client.select_clusters(where='gene', has=['CASTOR2'], genomic_modality='rna', p_value=0.05)
 >>> assert len(clusters_with_gene_set) > 0
 
->>> clusters_with_gene_set.get_list()[0].keys()
-dict_keys(['cluster_method', 'cluster_data', 'grouping_name', 'dataset'])
```

Comment:
Oh, yes, this will actually be able to be left in as is.  I meant to comment it out, rather than remove it, as this discrepancy will be resolved server-side.